apps/meteor/app/livechat/server/lib/Contacts.ts

```diff
@@ -187,42 +200,158 @@ export const Contacts = {
 	},
 };
 
-export function isSingleContactEnabled(): boolean {
-	// The Single Contact feature is not yet available in production, but can already be partially used in test environments.
-	return process.env.TEST_MODE?.toUpperCase() === 'TRUE';
+export async function getContactManagerIdByUsername(username?: IUser['username']): Promise<IUser['_id'] | undefined> {
+	if (!username) {
+		return;
+	}
+
+	const user = await Users.findOneByUsername(username, { projection: { _id: 1 } });
```

Comment:
Not sure if this accepts a type param, but if it does, pls

Reply:
Not really a big deal as the result is only used in the following line and not copied anywhere else, but I'll change it.